rosdep/python.yaml

```diff
@@ -991,6 +991,19 @@ python-expiringdict:
     pip:
       packages: [expiringdict]
   ubuntu: [python-expiringdict]
+python-face-recognition:
```

Comment:
Please use the suffix `-pip` for pip-only entries as described in the contribution guide.

Reply:
done